Changed Forms Style and Navbar effects

akshayks13 opened the pull request:
Description:
This PR addresses Issue #271 by improving the overall user experience through enhanced styling for the form and a smoother hover effect for the navigation bar. The following changes were implemented:

Changes Made:
Form Styling Enhancements:

Adjusted padding and border radius for better spacing and alignment.
Updated input fields and textarea to provide a more consistent look and feel across all devices.
Improved responsiveness by ensuring form elements resize gracefully on different screen sizes.
Added subtle animations to the submit button, including a hover effect that gives visual feedback when interacted with.
Navbar Hover Effect:

Introduced a smooth underline hover animation for the navigation links.
Enhanced link color transitions to create a more interactive and dynamic feel.
Improved contrast for better readability and accessibility.

akshayks13 commented:
Please add the Hacktoberfest and GSSoc badges

Altair-05 commented:
@akshayks13  plz resolve conflicts

Altair-05 commented:
@akshayks13, please resolve the conflicts and attach a screenshot/video showing the changes you made

akshayks13 commented:
> Description: This PR addresses Issue #271 by improving the overall user experience through enhanced styling for the form and a smoother hover effect for the navigation bar. The following changes were implemented:
> 
> Changes Made: Form Styling Enhancements:
> 
> Adjusted padding and border radius for better spacing and alignment. Updated input fields and textarea to provide a more consistent look and feel across all devices. Improved responsiveness by ensuring form elements resize gracefully on different screen sizes. Added subtle animations to the submit button, including a hover effect that gives visual feedback when interacted with. Navbar Hover Effect:
> 
> Introduced a smooth underline hover animation for the navigation links. Enhanced link color transitions to create a more interactive and dynamic feel. Improved contrast for better readability and accessibility.

<img width="1365" alt="Screenshot 2024-10-09 at 10 59 23 AM" src="https://github.com/user-attachments/assets/07287833-8130-4faa-8e72-
<img width="1385" alt="Screenshot 2024-10-09 at 10 59 49 AM" src="https://github.com/user-attachments/assets/7fbf7a21-5c26-487e-b04f-0c958108d17a">
7d2fc655c301">

Uploaded the screenshots and also requested my PR , I have made the necessary changes as I mentioned
so please review it .

dohinafs commented:
Kindly resolve the conflicts

akshayks13 commented:
> Kindly resolve the conflicts

I resolved the conflict and added the Screenshot as well

dohinafs commented:
> > Kindly resolve the conflicts
> 
> I resolved the conflict and added the Screenshot as well

Yes, but that was over a week ago. Kindly resolve the conflicts and I'll merge your PR as soon as possible.
